### PR TITLE
Templates: Added static DIY template

### PR DESCRIPTION
**Pull request by mariano-formidable**
## Summary

This PR adds a static version of the DIY template to our repo and updates Travel template with a few icons/images it was missing.

When testing this PR, be sure to pull it down and visit the specific template detail page to ensure that all the correct images/content load.

---

You should be able to navigate to the DIY template detail page by going to this url:
`/wp-admin/admin.php?page=stories-dashboard#/template-detail?id=3&isLocal=false`

image

The Travel template detail page url:
`/wp-admin/admin.php?page=stories-dashboard#/template-detail?id=7&isLocal=false`

image

### CODE REVIEW

File: assets/src/dashboard/app/views/templates/index.js [hunk 1]
```diff
@@ -151,6 +151,8 @@ function TemplatesGallery() {
             centerActionLabel={__('View details', 'web-stories')}
             bottomActionLabel={__('Use template', 'web-stories')}
             updateStory={() => {}}
+            duplicateStory={() => {}}
+            trashStory={() => {}}
```
Comment (BrittanyIRL) [inline assets/src/dashboard/app/views/templates/index.js]: will templates ever have these actions? If not maybe we should set them as optional and check they exist before setting them in the menu?  This sounds like it's out of scope, just a question!

**Review by BrittanyIRL — APPROVED** [review]
This looks so great and works exactly as you specified!

File: assets/src/dashboard/app/views/templates/index.js (hunk 1 above)
Comment (mariano-formidable) [reply assets/src/dashboard/app/views/templates/index.js]: Yeah I'm not sure 🤔 I just wanted the prop errors to go away.  I believe Carlos is working on (or going to work on) updating the template gallery UI to remove title and modify date from the cards, so I'mma let him figure out how best to handle these in the long term (I think they'll just be going away, they may not need a context menu like the normal stories 🤔).

File: assets/src/dashboard/app/views/shared/storyGridView.js [hunk 2]
```diff
@@ -115,14 +115,10 @@ const StoryGridView = ({
...
-                ? undefined
-                : {
-                    targetAction: story.centerTargetAction,
-                    label: centerActionLabel,
-                  }
-            }
+            centerAction={{
+              targetAction: story.centerTargetAction,
+              label: centerActionLabel,
+            }}
```
Comment (carloskelly13) [inline assets/src/dashboard/app/views/shared/storyGridView.js:121]: I think with this we get the Preview button again on the templates. I wasn't sure if we wanted that or not with the new designs.

**Review by carloskelly13 — APPROVED** [review]
I have one question that pertains to the gallery view. But it's likely a new vs old design discrepancy. If we want the Preview button again on the card to play the animations then this is good to merge!

File: assets/src/dashboard/app/views/shared/storyGridView.js (hunk 2 above)
Comment (mariano-formidable) [reply assets/src/dashboard/app/views/shared/storyGridView.js:121]: It just enables a middle link that we can call whatever we want.  I named it "View" for templates and it links to the template detail page which I think is part of the new designs too.
image

**Pull request merged**